test-avocado/lib/libjournal.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2013 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Tools for writing Cockpit test cases.
+"""
+
+import re       # journal matching
+import logging  # debug output in classes
+from avocado.utils import process # Journal output
+
+log=logging.getLogger("Journal")
+
+admins_only_pam = """
```

Comment:
for me it is more general (I can use it in more tests) so that that I've added it to library